### ISSUE TITLE
fetchSuccess receives response that has not been parsed

### ISSUE THREAD

**Issue by ijcd** [joneath/infiniScroll.js#7]
Backbone's collection.fetch seems to pass a non-parsed result objet to fetchSuccess. It should probably pass the parsed version since that is what is defined to return the array which infiniScroll calls response.length on...

**Comment by joshbedo** [comment]
Yeah just came across this same problem. Right now i have a nested collection that represents the top level length and since it's not being parsed the length is always the same when it's successful. This also means that infiniScroll never triggers the `success` method or `error` method because the `length` hasn't changed from the `lastLength`.
